tests/tlstest.py

```diff
@@ -568,12 +568,27 @@ def connect():
 
     synchro.recv(1)
     connection = connect()
-    # TODO add client certificate support in TLS 1.3
-    settings = HandshakeSettings()
-    settings.maxVersion = (3, 3)
-    connection.handshakeClientCert(x509Chain, x509Key, settings=settings)
+    connection.handshakeClientCert(x509Chain, x509Key)
```

Comment:
is the removal of settings= intentional?

Reply:
yes, no settings means use default settings, since we don't need to set `maxVersion` any more, the default == HandshakeSettings()